### PR TITLE
Update bedrock v4 docs

### PR DESCRIPTION
**Pull request by silv-io**
This PR updates the documentation to cover the new LLM-enabled functionality of Bedrock and the new configuration variables.

### CODE REVIEW

File: content/en/user-guide/aws/bedrock/index.md [hunk 1]
```diff
@@ -75,5 +86,5 @@ $ awslocal bedrock-runtime converse \
 
 ## Limitations
 
-* LocalStack Bedrock implementation is mock-only and does not run any LLM model locally.
+* LocalStack Bedrock currently only officially supports text-based models.
```
Comment (dfangl) [inline content/en/user-guide/aws/bedrock/index.md:89]: What does "currently only officially" mean exactly?

**Review by dfangl — APPROVED** [review]
LGTM, thanks for the fast turnaround!

File: content/en/user-guide/aws/bedrock/index.md (hunk 1 above)
Comment (silv-io) [reply content/en/user-guide/aws/bedrock/index.md:89]: That at this point in time (=currently) we only officially support text-based models as opposed to image or other kinds of binary data models

Comment (dfangl) [reply content/en/user-guide/aws/bedrock/index.md:89]: My question would be what kind of models are unofficially supported :sweat_smile: . It's fine for me now, this was just the first question popping in my head. For me they are either supported or not, but this phrasing leads me to believe other models might be unofficially supported :P

**Pull request merged**